scripts/in_container/verify_providers.py

```diff
@@ -254,6 +254,16 @@ class ProviderPackageDetails(NamedTuple):
     ),
 }
 
+# If provided parse USE_AIRFLOW_VERSION environment variable and construct tuple (major, minor, path)
```

Comment:
We only use 2.3 check now and there is always going to be only one version checked, so I'd say we should hard-coded 2.3 and at possibly verify if the USE_AIRFLOW_VERSION matches it so that we don't forget to update the hard-coded value (as we did before) WDYT?

Reply:
I would better try to make this command more consistent with other command in `breeze release-management verify-provider-packages`

https://github.com/apache/airflow/blob/af9143eacdff62738f6064ae7556dd8f4ca8d96d/dev/breeze/src/airflow_breeze/commands/release_management_commands.py#L484-L518

So if we change version here we do not need to change it again in `scripts/in_container/verify_providers.py`

https://github.com/apache/airflow/blob/af9143eacdff62738f6064ae7556dd8f4ca8d96d/.github/workflows/ci.yml#L757-L761